include/fw_info.h

```diff
@@ -230,8 +230,13 @@ struct __packed fw_info {
 	 */
 	u32_t valid;
 
+	/* Offset from this fw_info struct to the start of the next linked
+	 * image. If no other image is linked with this one, this value is 0.
+	 */
+	u32_t next_img_offset;
```

Comment:
I'm wondering if instead of this, we can change the size to fill the partition. We may still want to create a new (boolean) field that says whether or not this is the last image.

Reply:
I tried this out, and  it does not feel right to re-use the `size` field for a different purpose than the size of the firmware. Other users might rely on this information, and not be known in the semantics of it filling the partition.